lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -159,6 +159,44 @@
     }
   }
 
+  /* Start: Override style for CCX grade book */
+  .container {
```

Comment:
Is this class only used by the CCX display of the gradebook? If so, can we make that clearer, maybe by naming it `.ccx-container` ?